r/R/install-arrow.R

```diff
@@ -267,3 +268,9 @@ wslify_path <- function(path) {
   end_path <- strsplit(path, drive_expr)[[1]][-1]
   file.path(wslified_drive, end_path)
 }
+
+on_rosetta <- function() {
+  # make sure to suppress warnings and ignore the stderr so that this is silent where proc_translated doesn't exist
+  identical(tolower(Sys.info()[["sysname"]]), "darwin") &&
+    identical(suppressWarnings(system("sysctl -n sysctl.proc_translated", intern = TRUE, ignore.stderr = TRUE)), "1")
+}
```

Comment:
```suggestion
}

```

To appease the linter